package.json

```diff
@@ -14,7 +14,7 @@
     "test": "npm run test-development && npm run test-production && npm run test-builders && npm run test-global-build",
     "test-builders": "npm run build-webpack-test && npm run testee-builders",
     "test-development": "npm run testee",
-    "test-global-build": "npm run build && npm run testee-global-build",
+    "test-global-build": "npm run build && bundlesize && npm run testee-global-build",
```

Comment:
I tacked it onto this build because everything is already being built for these tests, but maybe there’s a better place / it should be split off into its own test.